react-app-drawer-component/justin-app/src/AppDrawer.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { FiMenu } from 'react-icons/fi';
+import './AppDrawer.css';
+
+export default function AppDrawer({ items }) {
+  const [menu, setMenu] = useState(false);
+  const [currentIndex, setCurrentIndex] = useState('');
```

Comment:
currentIndex sounds like is is an index into the `items` array. But its default value is a string. Shouldn't it be a number, like 0? Or should the name be changed?